include/util/typedefs.hpp

```diff
@@ -60,6 +60,8 @@ using NameID = std::uint32_t;
 using EdgeWeight = std::int32_t;
 using TurnPenalty = std::int16_t; // turn penalty in 100ms units
 
+static const std::size_t INVALID_SIZE_T = std::numeric_limits<std::size_t>::max();
```

Comment:
`INVALID_INDEX` would probably be a more fitting name, since that is what it is used to denote in the using code.